### PR TITLE
Sidetheme

### PR DESCRIPTION
**Pull request by KennyImarah**
Side Theme

### CODE REVIEW

File: angular-techDraft/src/app/testComponent/test.component.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<p>
+  Class project!
+</p>
```
Comment (JuliaDunayeva) [inline angular-techDraft/src/app/testComponent/test.component.html]: Kenny, please remove lines 1-3

Comment (JuliaDunayeva) [reply angular-techDraft/src/app/testComponent/test.component.html]: Actually, you need to remove all your test component

File: angular-techDraft/src/app/theme/theme.component.html [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<p>
+ Class Project Theme 4!
+</p>
+
```
Comment (ghost) [inline angular-techDraft/src/app/theme/theme.component.html]: Kenny remove line 1-3

**Review by KennyImarah — COMMENTED** [review]
issue resovled

**Review by JaredHook — APPROVED** [review]
Looks good, thanks Kenny

**Pull request merged**